configs/config_web_default_guardrails.yml

```
@@ -28,55 +28,64 @@ general:
+    base_url: "https://integrate.api.nvidia.com/v1"
     api_key: ${NVIDIA_API_KEY}
-    max_retries: 10
+    temperature: 0.2
+    top_p: 0.7
+    max_tokens: 32768
+    num_retries: 5
+    chat_template_kwargs:
+      enable_thinking: false
+
+
 
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)